hoot-core/src/main/cpp/hoot/core/io/OsmGeoJsonWriter.cpp

```diff
@@ -141,9 +148,12 @@ void OsmGeoJsonWriter::_writeGeometry(ConstNodePtr n)
 void OsmGeoJsonWriter::_writeGeometry(ConstWayPtr w)
 {
   const vector<long>& nodes = w->getNodeIds();
-  bool isPolygon = AreaCriterion().isSatisfied(w) ||
-                   (nodes.size() > 0 && nodes[0] == nodes[nodes.size() - 1]);
-  _writeGeometry(nodes, (isPolygon) ? "Polygon" : "LineString");
+  string geoType = "LineString";
+  if (_useTaskingManagerFormat)
+    geoType = "MultiPolygon";
+  else if (AreaCriterion().isSatisfied(w) || (nodes.size() > 0 && nodes[0] == nodes[nodes.size() - 1]))
+    geoType = "Polygon";
+  _writeGeometry(nodes, geoType);
```

Comment:
The Tasking Manager format uses `MultiPolygon`s instead of `Polygon`s.  Force that here.